src/render/tutorials/getting-started-with-knotx-stack/edge.html.md

```diff
@@ -104,19 +105,19 @@ fileSystemRepo = knotx.core.repository.filesystem
 You may see that Knot.x instance reloaded modules and `fileSystemRepo` is now one of available modules. **No restart required!**
 
 ```
-      Deployed httpRepo=java:io.knotx.repository.http.HttpRepositoryConnectorVerticle [c81f07ae-9345-482a-bd7f-af3e261876e0]
-      Deployed assembler=java:io.knotx.assembler.FragmentAssemblerVerticle [d8010ea9-6b65-482b-a37c-2139bf154413]
-      Deployed dataBridge=java:io.knotx.databridge.core.DataBridgeKnot [db7b7503-71e0-40e0-adde-9012c885d581]
-      Deployed splitter=java:io.knotx.splitter.FragmentSplitterVerticle [ed249e56-92c3-486d-827a-ad506a7e0ac3]
-      Deployed hbsKnot=java:io.knotx.knot.templating.HandlebarsKnotVerticle [b19ea9d2-59ed-4926-9365-3579af42895b]
-      Deployed fileSystemRepo=java:io.knotx.repository.fs.FilesystemRepositoryConnectorVerticle [0879f874-1276-44b9-b746-585ab19f7d25]
-      Deployed dataSourceHttp=java:io.knotx.databridge.http.HttpDataSourceAdapter [9caebb6a-18ed-474f-9182-56efdd180771]
-      Deployed server=java:io.knotx.server.KnotxServerVerticle [0c5f5136-c925-4f93-88b3-d24233a54988]
+    Deployed httpRepo=java:io.knotx.repository.http.HttpRepositoryConnectorVerticle [da4cfa69-04f2-4a79-9c63-39b27785c1e3]
+    Deployed assembler=java:io.knotx.assembler.FragmentAssemblerVerticle [b77754bc-b762-454a-b5bc-0a446c3db0d4]
+    Deployed dataBridge=java:io.knotx.databridge.core.DataBridgeKnot [e7aabad7-44a7-412f-ba69-dc82d6931409]
+    Deployed splitter=java:io.knotx.splitter.FragmentSplitterVerticle [fd9eee48-e6ae-407c-87e1-f3393f727ed3]
+    Deployed templateEngine=java:io.knotx.te.core.TemplateEngineKnot [606151e2-3971-4732-9eb9-60beff83ef1f]
+    Deployed dataSourceHttp=java:io.knotx.databridge.http.HttpDataSourceAdapter [17a6c160-2421-46d2-aca7-1d7f2e0ca71d]
+    Deployed server=java:io.knotx.server.KnotxServerVerticle [644ca0e5-1df6-49a7-a26f-d2f123f777f2]
+    Deployed fileSystemRepo=java:io.knotx.repository.fs.FilesystemRepositoryConnectorVerticle [0879f874-1276-44b9-b746-585ab19f7d25]
 ```
 
 Now, let's configure `fileSystemRepo` to read files from the local filesystem.
 
-Create `content` directory in `KNOTX_HOME` and put there following page template with Knot.x snippet (`<script data-knotx-knots="databridge,handlebars">...`):
+Create `content` directory in `KNOTX_HOME` and put there following page template with Knot.x snippet (`<script data-knotx-knots="databridge,te">...`):
```

Comment:
knotx:snippet

Reply:
That would require changing default `tagName` to `knotx:snippet` here:
https://github.com/Knotx/knotx-stack/blob/master/knotx-stack-manager/src/main/packaging/conf/application.conf#L30

Reply:
Discussed, we will make `knox:snippet` the official default tag in the next major release.